hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  if (parseInt(elem.value) >= 0) {
```

Comment:
The assignment sheet wanted us to check for non-number input. Probably want to add a check for that in this function.

Reply:
Thank you, I overlooked that. Will change that shortly.

Reply:
I have added the checks for non-number inputs.